### PR TITLE
Add 'Reviewing annotations in Log Detective'

### PR DESCRIPTION
**Pull request by TomasTomecek**
I'd appreciate if I should cover more things in this post.

### CODE REVIEW

File: _posts/2024-08-16-logdetective-reviews.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: "Reviewing annotations in Log Detective"
+author: "Tomas Tomecek"
+layout: "post"
+---
+
+We appreciate everyone who submitted annotations, thank you! It will allow us to
+create a model with knowledge about building software, not just RPMs, but also
+container images, test failures and various language specific buildsystems. I
+envision that Log Detective will provide useful answers to most of logs with
+build failures. Though we still have a long way getting there.
```
Comment (xsuchy) [inline _posts/2024-08-16-logdetective-reviews.md:11]: I don't expect all Copr users to immediately know what Log Detective is.

I suggest to edit the first paragraph to:

```
You may have heard of our [LogDetective](https://logdetective.com/) project.
We want to use AI to process your log of failed builds, show you where
the problem is and how to fix it. We have enough failure logs, but we need
to supplement them with notes on why the build went wrong and how to fix it. 
```

File: _posts/2024-08-16-logdetective-reviews.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## How do I review snippets?
+
+Once you click the "Review logs" button at the top menu, you'll reach the new
+review interface. The instructions are on the left side.
+
+![](/assets/img/posts/logdetective-review-instructions.png)
+
+You are welcome to add more snippets if you can spot more information that
+should be annotated. If an annotation is incorrect, please downvote it and
+create a new one.
```
Comment (FrostyX) [inline _posts/2024-08-16-logdetective-reviews.md:40]: I don't know if to mention it in this paragraph or a different one but if a snippet is correctly highlighted only the comment is incorrect, it is possible to edit the comment.

Comment (TomasTomecek) [reply _posts/2024-08-16-logdetective-reviews.md:40]: oh, very good point!

File: _posts/2024-08-16-logdetective-reviews.md (hunk 1 above)
Comment (TomasTomecek) [reply _posts/2024-08-16-logdetective-reviews.md:11]: thanks, will add that!

**Comment by FrostyX** [comment]
+1, before merging, please rebase and squash all the commits together.

**Comment by TomasTomecek** [comment]
rebased & squashed now

**Review by xsuchy — APPROVED** [review]
+1

**Pull request merged**